### PR TITLE
Gives Soviet and Capitalist Golems a antag datum

### PR DESCRIPTION
**Pull request by fluffe9911**
Currently the removal and huds are currently broken for some unknown reason nich just told me to go ahead and post it so he can look at it when he wakes up honk

### CODE REVIEW

**Comment by boodaliboo** [comment]
i am

File: code/modules/antagonists/golems/capitalist.dm [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	message_admins("[key_name_admin(admin)] made [key_name_admin(new_owner)] into [name].")
+	log_admin("[key_name(admin)] made [key_name(new_owner)] into [name].")
+
+/datum/antagonist/capitalist/admin_remove(datum/mind/new_owner,mob/admin)
+	new_owner.remove_antag_datum(/datum/antagonist/capitalist)
+
+/datum/antagonist/capitalist/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_capitalist_icons_added(M)
+	M.set_species(/datum/species/golem/soviet)
```
Comment (Bibby0110) [inline code/modules/antagonists/golems/capitalist.dm]: :thonk:

Comment (fluffe9911) [reply code/modules/antagonists/golems/capitalist.dm]: Capitalism is a myth honk

**Comment by Bibby0110** [comment]
y

**Pull request closed without merging**